python/lib/sift_py/data_import/ch10.py

```diff
@@ -79,17 +86,15 @@ def upload_ch10(
         Override `run_name` to specify the name of the run to create for this data. Default is None.
         Override `run_id` to specify the id of the run to add this data to. Default is None.
         """
-        # Trigger the first packet read to get the row headers
-        # and csv_config_data_columns.
-        next(ch10_file)
+        ch10_file.initialize_csv_data_columns()
 
         assert getattr(
             ch10_file, "csv_config_data_columns"
         ), "`csv_config_data_columns` was not set correctly on the first iteration"
 
         config_info: Dict[str, Any] = {
             "asset_name": asset_name,
-            "first_data_row": 1,
+            "first_data_row": 2,
```

Comment:
Since we are not sending the entire file instead of skipping the header, this should be 1.